imageRecognition/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:14.04
+
+MAINTAINER Ria Bhatia
+
+COPY faceDetection.py /
+COPY images.tar.gz
```

Comment:
Need to add "/" at the end of the line so COPY knows where to copy images.tar.gz